airflow/www/views.py

```diff
@@ -751,22 +755,27 @@ def log(self):
                     ti_execution_completed or ti_ran_more_than_once)
 
                 # S3
-                if remote_log_path.startswith('s3:/'):
-                    remote_log += log_utils.S3Log().read(
-                        remote_log_path, return_error=surface_log_retrieval_errors)
+                if remote_log_dir.startswith('s3:/'):
```

Comment:
Does this work as expected there is currently a running task that is not the first try that is writing to local disk and hasn't written to AWS yet? Seems like this would fail reading the last try number and not set remote_log_loaded to True when it should have.